getting-started/vercel.mdx

```diff
@@ -0,0 +1,152 @@
+---
+title: Vercel
+description: Implementing Edgee in front of Vercel for better performance and SEO control
+icon: stairs
```

Comment:
I would add the Vercel logo here (it should be possible to reference a local svg file)

If not, I would rather have no logo at all (similar to what we do with the components)